arm-network/2017-03-01/swagger/applicationGateway.json

```diff
@@ -419,12 +444,13 @@
         },	  
         "health": {
           "type": "string",
-          "description": "Health of backend server. Possible values are: 'Unknown', 'Up', 'Down', and 'Partial'.",
+          "description": "Health of backend server.",
```

Comment:
It get's automatically added to the auto-generated code. Otherwise the possible enum values would be twice in a row in the documentation of the property.